src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+
+#### Background reading
+
+[Robist Incentives Group: PoS incentives](https://ethereum.github.io/beaconrunner/)
+
+#### Recent research
+
+[Vitalik on PBS](https://notes.ethereum.org/s3JToeApTx6CKLJt8AbhFQ#Hybrid-PBS-can-we-use-proposers-only-for-inclusion-of-last-resort)
+
+[Three Attacks on PoS Ethereum](https://arxiv.org/abs/2110.10086)
+
+[Low-cost attacks on Ethereum 2.0 by sub 1/3 stakeholders](https://arxiv.org/abs/2102.02247)
+
+
+### Liquid staking and derivatives
+
+Liquid staking allows users with less than 32 ETH to receive staking yields by swapping ether for a token representing staked ether that can be used in DeFi. However, the incentives and market dynamics associated with liquid staking as well as its effect on Ethereum's security (e.g. centralization risks).
+
+#### Background reading
+
+[Ethresear.ch liquid staking](https://ethresear.ch/search?q=liquid%20staking)
+
+
+#### Recent research
+
+[Handling withdrawals from Lido](https://ethresear.ch/t/handling-withdrawals-in-lidos-eth-liquid-staking-protocol/8873)
+
+[Withdrawal credentials](https://ethresear.ch/t/withdrawal-credential-rotation-from-bls-to-eth1/8722)
+
+### DeFi
+
+Decentralized finance (DeFi) is one of the primary classes of application built on top of Ethereum. DeFi aims to create composable "money legos" that allow users to store, transfer, lend, borrow and and invest crypto-assets using smart contracts. This is a fast-moving space that is constantly updating. Research into secure, efficient and accessible protocols is constantly needed.
+
+#### Background reading
+
+[DeFi](https://ethereum.org/en/defi/)
+
+[Coinbase: What is DeFi?](https://www.coinbase.com/learn/crypto-basics/what-is-defi)
+
+
+#### Recent research
+
+[Decentralized finance, centralized ownership?](https://arxiv.org/pdf/2012.09306.pdf)
+
+[Optimism: The road to sub-dollar transactions](https://medium.com/ethereum-optimism/the-road-to-sub-dollar-transactions-part-2-compression-edition-6bb2890e3e92)
+
+### Fee markets
+
+Ethereum transaction fees protect the network from spam, denial-of-service attacks and bad smart-contract code. However, it also affects the end-user by influencing the cost of interacting with Ethereum. Balancing network security with end-user user-experience is an ongoing challenge that requires further research and development. 
+
+#### Background reading
+
+[EIP-1559 (Robust Incentives Group)](https://ethereum.github.io/abm1559/notebooks/eip1559.html)
+
+#### Recent research
+
+[EIP-4396](https://eips.ethereum.org/EIPS/eip-4396)
+
+[Multidimensional EIP-1559](https://ethresear.ch/t/multidimensional-eip-1559/11651)
+
+[Making EIP-1559 more like an AMM curve](https://ethresear.ch/t/make-eip-1559-more-like-an-amm-curve/9082)
+
+[Cross domain MEV](http://arxiv.org/abs/2112.01472)
+
+
+## Clients and protocol research
```

Comment:
Barnabe made a suggestion here that I liked - breaking out protocol research vs. development into separate categories.

e.g. page layout:

```
## General research resources

## Protocol research

### Consensus

- e.g. PoS consensus, fork choice, single slot finality, weak subjectivity

### Execution

- e.g. statelessness

## Protocol development

### Consensus clients

- client specs
- link to clients ( https://ethereum.org/en/developers/docs/nodes-and-clients/#consensus-clients)

### Execution clients

- client specs
- link to clients (https://ethereum.org/en/developers/docs/nodes-and-clients/#execution-clients)
```

Reply:
More notes on that here: https://notes.ethereum.org/wvg_V07BTPiSmg2AoUM2_Q?view#Open-research-content